src/contrib/cluster/Akka.Cluster.Sharding/Internal/EventSourcedRememberEntitiesProvider.cs

```diff
@@ -39,7 +48,14 @@ public Props ShardStoreProps(string shardId)
         /// <returns></returns>
         public Props CoordinatorStoreProps()
         {
-            return EventSourcedRememberEntitiesCoordinatorStore.Props(TypeName, Settings);
+            var backoffOptions = Backoff.OnStop(
```

Comment:
Wraps the `RememberEntitiesCoordinator` inside a `BackoffSupervisor` - uses the same tuning parameters as the `ShardingCoordinator` for backoff.